guides/17_Migration_to_3_0.md

```diff
@@ -8,6 +8,12 @@ Installation involves adjusting your `package.json` to depend on version `3.0.0`
 npm install --save amazon-chime-sdk-js@3
 ```
 
+Note that, currently only pre-release NPM versions of `3.0.0` are available until we do the final major version release. Do the following step to install the latest beta version for `amazon-chime-sdk-js`:
+
+```shell
+npm install --save amazon-chime-sdk-js@beta
```

Comment:
Is the `--save` needed?

Reply:
Just copy pasted from installation but added beta 😅 . Though the --save is not required anymore as it is the default while doing NPM installs starting NPM v5.